src/gaurav/BankSystem.java

```diff
@@ -0,0 +1,77 @@
+package gaurav;
+
+public class BankSystem {
+		String name;
+		int totalBalance;
+		int debit=0;
+		static int totalDebitCount=0;
+		int credit=0;
+		static int totalCreditCount=0;
+		int printBalance=0;
+		static int totalPrintBalance=0;
+		
+		void setData(int initialBalance,String custName) 
+		{
+			totalBalance=initialBalance;
+			name=custName;
+		}
+		void debitAmount(int amount) 
+		{
+			if(amount<=totalBalance)
+			{
+				totalBalance=totalBalance-amount;
+				debit++;
+				totalDebitCount++;
+			}
+		}
+		
+		void creditAmount(int amount)
+		{
+			if(amount<=totalBalance) 
```

Comment:
Why this require here?
while credits esa thodi hoga ki like If you have 2000 in ac and some want to x'fer 20000 in ac then he cannot x'fer to your acc.